Data/GraphViz/Types/Graph.hs

```diff
@@ -231,9 +231,9 @@ emptyGA = GA S.empty S.empty S.empty
     dg' = addNode n mc as dg
 
     merge = addSuccRev n ps'' . addPredRev n ss''
-            -- ^ Add reverse edges
```

Comment:
Whilst these aren't processed by Haddock, I prefer to keep the carets here to point to the line the comment is referring to (making it a bit more obvious what I'm talking about).